src/main/java/com/eternalcode/combat/drop/DropInfo.java

```diff
@@ -0,0 +1,95 @@
+package com.eternalcode.combat.drop;
+
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DropInfo {
```

Comment:
`DropInfo` mało mówi, jest to zbyt ogólna nazwa + jest mutable (Info z reguły jest immutable). Ja bym tutaj nazwał to po prostu `Drop`